core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/click/ClickModifierTest.java

```diff
@@ -1,14 +1,14 @@
 /**
  * Automated Exploratory Tests
- *
+ * <p>
```

Comment:
Other license headers don't have those `<p>` tags. do we really need them?
Could they be same in all files? Maybe done within some other task if it is needed? (must have)